src/gui/Hud.cpp

```diff
@@ -1105,15 +1105,15 @@ void ActiveSpellsGui::init() {
 	m_slotSize = Vec2f(24.f, 24.f);
 	m_spacerSize = Vec2f(60.f, 50.f);
 	m_slotSpacerSize = Vec2f(0.f, 9.f);
-	m_ucFlick = 0;
+	m_ucFlick = false;
 }
 
 void ActiveSpellsGui::update(Rectf parent) {
 	
 	float intensity = 1.f - PULSATE * 0.5f;
 	intensity = glm::clamp(intensity, 0.f, 1.f);
 
-	m_ucFlick++;
+	m_ucFlick = !m_ucFlick;
```

Comment:
Would be nice to change this at a fixed rate rater than per frame in the future.